components/Dialogs/tests/unit/MDCAlertControllerCustomizationTests.m

```diff
@@ -20,43 +20,44 @@
 #import <XCTest/XCTest.h>
 
 @interface MDCAlertControllerCustomizationTests : XCTestCase
+@property(nonatomic, nullable, strong) MDCAlertController *alert;
+@property(nonatomic, nullable, weak) MDCAlertControllerView *alertView;
+@property(nonatomic, nullable, weak) MDCDialogPresentationController *presentationController;
 
 @end
 
 @implementation MDCAlertControllerCustomizationTests
 
-MDCAlertController *alert;
-MDCAlertControllerView *alertView;
-
 - (void)setUp {
   [super setUp];
-  alert = [MDCAlertController alertControllerWithTitle:@"Title" message:@"Message"];
-  alertView = (MDCAlertControllerView *)alert.view;
+  self.alert = [MDCAlertController alertControllerWithTitle:@"Title" message:@"Message"];
+  self.alertView = (MDCAlertControllerView *)self.alert.view;
```

Comment:
Tip: Use `- [MDCAlertController.alertView]` to safely access the `MDCAlertControllerView` property. If you don't want to expose it in a testing category, please perform a type check `-isKindOfClass:` first.

Reply:
Added the type check.